macaroonbakery/bakery/_checker.py

```diff
@@ -111,21 +111,18 @@ def _init(self, ctx):
             if not self._executed:
                 self._init_once(ctx)
                 self._executed = True
-        if self._init_errors:
-            raise AuthInitError(self._init_errors[0])
 
     def _init_once(self, ctx):
         self._auth_indexes = {}
         self._conditions = [None] * len(self._macaroons)
         for i, ms in enumerate(self._macaroons):
             try:
-                ops, conditions = self.parent._macaroon_opstore.macaroon_ops(
-                    ms)
-            except VerificationError:
-                raise
-            except Exception as exc:
-                self._init_errors.append(exc.args[0])
+                ops, conditions = self.parent._macaroon_opstore.macaroon_ops(ms)
+            except VerificationError as e:
+                self._init_errors.append(str(e))
                 continue
+            except Exception as exc:
+                raise AuthInitError(exc.args[0])
```

Comment:
I think you can just `raise AuthInitError(str(exc))` here, so it's clearer you're passing the exception message

Reply:
done